exercises/01-getting-started.md

```diff
@@ -1,63 +1,81 @@
 # Getting started
-In the following we describe how to get started with this project.
+
 You will create a new **private** repository based on this repository and run the application locally to get familiar with it.
 
 > In the original session, the participants were handed out a participantId. If you go through the tasks on your own, make up your own id. We recommend to refrain from using personal data.
 
 ## Create a new repository based on this template
-Click the *Use this template* button on the top of the GitHub page.
+
+Click `Use this template` on the top of the GitHub page.
```

Comment:
According to [our style guide](https://github.com/SAP/cloud-sdk/blob/main/STYLEGUIDE.md#use-bold-format-for-names), buttons, labels etc should not use the 'code' style (backticks) but be bold (two asterisks `**Use this template**`)

Reply:
apply the style guide